### ISSUE TITLE
LocationError after updating the code

### ISSUE THREAD

**Issue by hvelarde** [collective/collective.revisionmanager#10]
After updating the code to use the master branch I get the following error while trying to access the control panel configlet:

```python
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module z3c.form.form, line 239, in __call__
  Module z3c.form.form, line 163, in render
  Module Products.Five.browser.pagetemplatefile, line 125, in __call__
  Module Products.Five.browser.pagetemplatefile, line 59, in __call__
  Module zope.pagetemplate.pagetemplate, line 132, in pt_render
  Module five.pt.engine, line 93, in __call__
  Module z3c.pt.pagetemplate, line 149, in render
  Module chameleon.zpt.template, line 258, in render
  Module chameleon.template, line 191, in render
  Module chameleon.template, line 171, in render
  Module 97c37ab302ad62e3bdf0a88e365b7223, line 1522, in render
  Module 7731075829d68e051794cd3fb8257b08, line 283, in render_master
  Module db1e64eae578bee90603533e304f2d63, line 1425, in render_master
  Module 7731075829d68e051794cd3fb8257b08, line 269, in __fill_content
  Module 97c37ab302ad62e3bdf0a88e365b7223, line 801, in __fill_prefs_configlet_content
  Module five.pt.expressions, line 154, in __call__
  Module five.pt.expressions, line 126, in traverse
  Module zope.traversing.adapters, line 136, in traversePathElement
   - __traceback_info__: ({'deleted_average': '7.1', 'time': '136.46', 'deleted_versions': 558, 'total_average': '8.1', 'total_versions': 9484, 'exisiting_histories': 1087, 'existing_versions': 8926, 'deleted_histories': 79, 'total_histories': 1166, 'existing_average': '8.2'}, 'existing_histories')
  Module zope.traversing.adapters, line 50, in traverse
   - __traceback_info__: ({'deleted_average': '7.1', 'time': '136.46', 'deleted_versions': 558, 'total_average': '8.1', 'total_versions': 9484, 'exisiting_histories': 1087, 'existing_versions': 8926, 'deleted_histories': 79, 'total_histories': 1166, 'existing_average': '8.2'}, 'existing_histories', ())
LocationError: ({'deleted_average': '7.1', 'time': '136.46', 'deleted_versions': 558, 'total_average': '8.1', 'total_versions': 9484, 'exisiting_histories': 1087, 'existing_versions': 8926, 'deleted_histories': 79, 'total_histories': 1166, 'existing_average': '8.2'}, 'existing_histories')

 - Expression: "summaries/existing_histories"
 - Filename:   ... /collective/revisionmanager/browser/revisionssettings.pt
 - Location:   (line 86: col 41)
 - Source:     ... td tal:content="summaries/existing_histories">existing_histo ...
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - Arguments:  repeat: {...} (0)
               template: <ViewPageTemplateFile - at 0x7f3cea3c1cd0>
               views: <ViewMapper - at 0x7f3cde4abcd0>
               modules: <instance - at 0x7f3cf7bdb908>
               args: <tuple - at 0x7f3cfefaa050>
               here: <ImplicitAcquisitionWrapper Plone at 0x7f3cde4a3eb0>
               user: <ImplicitAcquisitionWrapper - at 0x7f3cde492050>
               nothing: <NoneType - at 0x91a870>
               container: <ImplicitAcquisitionWrapper Plone at 0x7f3cde4a3eb0>
               request: <instance - at 0x7f3cde420488>
               wrapped_repeat: <SafeMapping - at 0x7f3cde2c6e68>
               options: {...} (0)
               traverse_subpath: <list - at 0x7f3cde6af050>
               default: <object - at 0x7f3cfeec5500>
               loop: {...} (2)
               context: <ImplicitAcquisitionWrapper Plone at 0x7f3cde4a3eb0>
               view: <RevisionsControlPanel revisions-controlpanel at 0x7f3cde364950>
               translate: <function translate at 0x7f3cde48c578>
               root: <ImplicitAcquisitionWrapper Zope at 0x7f3cde3456e0>
               target_language: en
```

**Comment by tschorr** [comment]
can you try reinstalling the add on please, I think a typo fix is causing this (exisisting->existing)

**Comment by tschorr** [comment]
if that works, I'll add an upgrade step to clear the cache.

**Comment by hvelarde** [comment]
reinstalling works so an upgrade step is needed; thank you!

can we have a new release after that? don't forget to include the `.mo` files on the egg; are you using [zest.releaser](https://pypi.python.org/pypi/zest.releaser)?

**Comment by tschorr** [comment]
I've just release 0.7. The `.mo` files are included - they were also already included in the 0.6 release, is there anything wrong with the translations?

**Comment by hvelarde** [comment]
thanks! I'll test and let you know in case of any issues.
